user_tools/src/spark_rapids_pytools/resources/azure-instances-catalog.json

```diff
@@ -0,0 +1,47 @@
+{
+  "Standard_D1_v2": {
+      "VCpuInfo": {
+      "DefaultVCpus": 1
+      },
+      "MemoryInfo": {
+      "SizeInMiB": 3584
+      }
+  },
+  "Standard_DS3_v2": {
+    "VCpuInfo": {
+    "DefaultVCpus": 4
+    },
+    "MemoryInfo": {
+    "SizeInMiB": 14336
+    }
+  },
+  "Standard_DS4_v2": {
+    "VCpuInfo": {
+    "DefaultVCpus": 8
+    },
+    "MemoryInfo": {
+    "SizeInMiB": 28672
+    }
+  },
+  "Standard_NC6s_v3": {
+    "VCpuInfo": {
+      "DefaultVCpus": 6
+    },
+    "MemoryInfo": {
+      "SizeInMiB": 114688
+    },
+    "GpuInfo": {
+      "Gpus": [
+        {
+          "Name": "V100",
```

Comment:
We don't support V100 in the wrapper-tools.
Do they offer any other GPUs?

Reply:
Yes, they do.